<a href="https://colab.research.google.com/github/RiteshopShrivastava/Assignment-1-Q24-Basic-Statistics-level-1/blob/main/Assignment_1_Q24_Basic_Statistics_level_1.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

Q 24) A Government company claims that an average light bulb lasts 270 days. A researcher randomly selects 18 bulbs for testing. The sampled bulbs last an average of 260 days, with a standard deviation of 90 days. If the CEO's claim were true, what is the probability that 18 randomly selected bulbs would have an average life of no more than 260 days

In [7]:
from scipy import stats
import numpy as np

Pop mean =270 days, Sample mean = 260 days, Sample SD = 90 days, Sample size n = 18 bulbs, df = n-1 = 17

In [ ]:
# Null hypothesis Ho = average life of bulb >= 260 days
# Alternate hypothesis Ha = avaerage life of bulb < 260 days

Since the sample size is small (less than 30), and population SD is not known, therefore we use t-distribution

In [8]:
# find t-scores at x=260; t=(s_mean-P_mean)/(s_SD/sqrt(n))
t=(260-270)/(90/np.sqrt(18))
t

-0.4714045207910317

we can calculate P(X>=260) in two ways

1. By cummulative distribution function cdf
2. By Survival function sf

In [10]:
# using survival function ******sf=(1-cdf)}******

# p_value=stats.t.sf(abs(t_score),df=n-1)
p_value=stats.t.sf(abs(-0.4714),df=17)
print('Probability value for x>=260 is :',p_value)

Probability value fo x>=260 is : 0.32167411684460556


Since the p-value (.32) is greater than alpha(0.05), so accept null hypothesis. Therfore average life of a bulb is >= 260 days is true